### PR TITLE
migrate shadow hand from wiki.ros.org/Robots/Backup

### PR DESCRIPTION
**Pull request by wjwwood**

### CODE REVIEW

File: _posts/2017-05-02-shadow-hand.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+main-class: 'component'
+color: '#4F224B'
+tags:
+- 'manipulator'
+twitter_text: Checkout the Shadow Hand
+website: http://www.shadowrobot.com/products/dexterous-hand/
+wiki_homepage: http://wiki.ros.org/Robots/Shadow_Hand
+---
+
+TODO Fill in the long description/article here.
```
Comment (tfoote) [inline _posts/2017-05-02-shadow-hand.md]: There's still a TODO here for the main body content we usually copy this content from the wiki for the migration.

File: _posts/2017-05-02-shadow-hand.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Shadow Hand
+date: 2017-05-02
+description:
+icon: /assets/img/robots/shadow-hand/shadow_hand.jpg
+image: /assets/img/robots/shadow-hand/shadow_hand_rubiks.png
+introduction: The Shadow Dexterous Hand is a truly anthropomorphic approach to robot manipulation. With 20 actuated degrees of freedom, position and force sensors, and ultra sensitive touch sensors on the fingertips, the hand provides unique capabilities for problems that require the closest approximation of the human hand currently possible. The Shadow Dexterous Hand uses industry standard interfaces and can be used as a tele-operation tool or mounted on a range of robot arms as part of a robot system.
```
Comment (tfoote) [inline _posts/2017-05-02-shadow-hand.md]: We've been usually sticking to a 1 or two sentence overview here which gets rendered into the small box view.

**Comment by wjwwood** [comment]
Thanks, I addressed the comments.

**Comment by wjwwood** [comment]
Hang on, I have an unrelated change in there.

**Comment by wjwwood** [comment]
Ok, rebased.

**Pull request merged**